index.html

```diff
@@ -252,10 +293,19 @@ <h1 class="text-2xl font-semibold tracking-[7px] text-blue-0">TODO</h1>
         </div>
       </div>
     </main>
-    <footer class="mb-8 mt-16 flex justify-center font-josefin text-sm font-bold">
+    <footer class="mb-8 mt-16 flex flex-col items-center font-josefin text-sm font-bold">
       <p>Drag and drop to reorder list</p>
+      <div>
+        <button
+          type="button"
+          class="mt-2 bg-cyan-500 p-2 text-white hover:bg-slate-400 hover:text-black active:translate-y-1 active:bg-slate-500"
+          id="logout-btn"
+        >
+          LOG OUT
+        </button>
+      </div>
     </footer>
 
-    <script type="module" src="src/js/main.js"></script>
+    <script type="module" src="src/js/auth.js"></script>
```

Comment:
I guess you will still use the main.js, just that your auth logic will reside in the auth.js file